internal/core/src/segcore/segment_c.h

```diff
@@ -120,6 +120,12 @@ AddFieldDataInfoForSealed(CSegmentInterface c_segment,
                           CLoadFieldDataInfo c_load_field_data_info);
 
 //////////////////////////////    interfaces for SegmentInterface    //////////////////////////////
+CStatus
+ExistPk(CSegmentInterface c_segment,
+        const uint8_t* raw_ids,
+        const uint64_t size,
+        bool* results);
```

Comment:
bool* ?

Reply:
yes it queries for an ID batch, the `bool* results` points to a bool array